src/output.c

```diff
@@ -279,3 +280,36 @@ cmd_outputset(gcc_unused int argc, char **argv, struct mpd_connection *conn)
 	my_finishCommand(conn);
 	return 0;
 }
+
+int
+cmd_moveoutput(int argc, char **argv, struct mpd_connection *conn)
+{
+    char *name = argv[0];
+    int index;
+    if (parse_int(name, &index)) {
+        /* We decrement by 1 to make it natural to the user. */
+        index--;
+
+        // Look up the name for the index.
+        if (!mpd_send_outputs(conn)) {
+            printErrorAndExit(conn);
+        }
+
+        struct mpd_output *output;
+        while ((output = mpd_recv_output(conn)) != NULL) {
+            if (mpd_output_get_id(output) == (unsigned)index) {
+                name = mpd_output_get_name(output);
+                my_finishCommand(conn);
+                break;
+            }
+
+            mpd_output_free(output);
+        }
```

Comment:
Please move this to a function, e.g. `output_id_to_name()`; this smells like it could be reused some day.

Reply:
I did consider this, but ended up inlining it because `moveoutput` is the only command in the MPD protocol that uses the output name instead of an ID, so I thought it unlikely that it'd be needed elsewhere.

I'll move it if you feel strongly that it should be separate, or there's some output command I missed that needs index->name lookup.